rocky/rocky/views/ooi_list.py

```diff
@@ -142,6 +143,37 @@ def _set_scan_profiles(self, selected_oois: List[Reference], request: HttpReques
         )
         return self.get(request, *args, **kwargs)
 
+    def _set_oois_to_inherit(
+        self, selected_oois: List[Reference], request: HttpRequest, *args, **kwargs
+    ) -> HttpResponse:
+        for ooi in selected_oois:
```

Comment:
(Note) Keep in mind that this will stop at the first error while handling the oois. An alternative could be to process them all and keep a list of errors.